### DISCUSSION TITLE
Why did 0.26.0 remove React Native example app

### DISCUSSION THREAD

**Discussion by MaruanAlush** [callstack/react-native-builder-bob#320] [Q&A]
Hey everyone, I am curious as to why since 0.26.0 I am forced to use Expo as the example app and can't use pure React native (like it was possible to choose between the two in 0.25.0)?

**Comment by satya164** [comment]
With recent versions of Expo, you can use third-party native modules with a custom dev client - which was the main use case for having a pure React Native example app for a JS library.

If that's not enough and you need access to native code, you can also run `expo prebuild` inside the `example` folder to get the `android` and `ios` folders.
